packages/x-tree-view/src/internals/models/helpers.ts

```diff
@@ -8,6 +8,14 @@ export type DefaultizedProps<
   Required<Pick<P, RequiredProps>> &
   AdditionalProps;
 
+type IsAny<T> = 0 extends 1 & T ? true : false;
```

Comment:
NICE

Reply:
Found that one on Stackoverflow :laughing:

Reply:
I had to read an explanation article about it to understand that! :D